sdk/storage/azure-storage-blob/azure/storage/blob/_shared_access_signature.py

```diff
@@ -308,15 +308,17 @@ def get_token(self):
 
 
 def generate_account_sas(
```

Comment:
Two comments for the overall change:

- Can we add one test that generates an account SAS with non-default services (maybe Blob and File together) just to make sure it works. Also, if we support `str` (per my other comment), test that too.
- Can you update the CHANGELOG for all the packages.